CreateUserDataProcessing/handler.ts

```diff
@@ -0,0 +1,112 @@
+import * as express from "express";
+
+import { Context } from "@azure/functions";
+
+import { isLeft } from "fp-ts/lib/Either";
+
+import {
+  IResponseErrorConflict,
+  IResponseErrorValidation,
+  IResponseSuccessJson,
+  ResponseErrorFromValidationErrors,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+import { FiscalCode } from "italia-ts-commons/lib/strings";
+
+import {
+  IResponseErrorQuery,
+  ResponseErrorQuery
+} from "io-functions-commons/dist/src/utils/response";
+
+import { ContextMiddleware } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import { FiscalCodeMiddleware } from "io-functions-commons/dist/src/utils/middlewares/fiscalcode";
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "io-functions-commons/dist/src/utils/request_middleware";
+
+import { UserDataProcessingChoiceRequest } from "io-functions-commons/dist/generated/definitions/UserDataProcessingChoiceRequest";
+import { UserDataProcessingStatusEnum } from "io-functions-commons/dist/generated/definitions/UserDataProcessingStatus";
+import {
+  makeUserDataProcessingId,
+  UserDataProcessing,
+  UserDataProcessingModel
+} from "io-functions-commons/dist/src/models/user_data_processing";
+import { UserDataProcessingChoiceMiddleware } from "../utils/middlewares/userDataProcessing";
+
+/**
+ * Type of an CreateProfile handler.
+ */
+type ICreateUserDataProcessingHandler = (
+  context: Context,
+  fiscalCode: FiscalCode,
+  userDataProcessingChoiceRequest: UserDataProcessingChoiceRequest
+) => Promise<
+  // tslint:disable-next-line: max-union-size
+  | IResponseSuccessJson<UserDataProcessing>
+  | IResponseErrorValidation
+  | IResponseErrorQuery
+  | IResponseErrorConflict
+>;
+
+export function CreateUserDataProcessingHandler(
```

Comment:
since we're calling createOrUpdateByNewOne this should be:

```suggestion
export function UpsertUserDataProcessingHandler(
```

(same for file names)